recipe/meta.yaml

```diff
@@ -11,19 +11,19 @@
 # In this case, libstdcxx last changed the default version of an ELF symbol
 # in v12, therefore this is the lowest we can go for GCC <=12, see #97 or
 # https://github.com/gcc-mirror/gcc/commit/9e18a25331fa25c3907249fede65a02c6817b06e
-{% set last_symbol_bump = 12 %}
+{% set last_symbol_bump = 13 %}
```

Comment:
This comment is now outdated. What symbol did get a default version bump in GCC 13? https://gcc.gnu.org/onlinedocs/libstdc++/manual/api.html only lists additions for GCC 13.

Reply:
Fixed in #113